# provide the EvD node ID for the root of the AST to run.
# If null, the program is assumed as the root UUID to run.
string uuid
---
bool status
string message